addons/website/static/src/components/configurator/configurator.js

```diff
@@ -28,27 +26,72 @@ const WEBSITE_PURPOSES = {
     4: {id: 4, label: _lt("inform customers"), name: 'inform_customers'}
 };
 
-const hex2lab = (hex) => {
-    let r = parseInt(hex.substring(1, 3), 16) / 255, g = parseInt(hex.substring(3, 5), 16) / 255, b = parseInt(hex.substring(5, 7), 16) / 255, x, y, z;
-    [r, g, b] = [r, g, b].map((x) => (x > 0.04045) ? Math.pow((x + 0.055) / 1.055, 2.4) : x / 12.92);
-    x = (r * 0.4124 + g * 0.3576 + b * 0.1805) / 0.95047;
-    y = (r * 0.2126 + g * 0.7152 + b * 0.0722) / 1.00000;
-    z = (r * 0.0193 + g * 0.1192 + b * 0.9505) / 1.08883;
-    [x, y, z] = [x, y, z].map((x) => (x > 0.008856) ? Math.pow(x, 1 / 3) : (7.787 * x) + 16 / 116);
-    return [(116 * y) - 16, 500 * (x - y), 200 * (y - z)];
-};
+const PALETTE_NAMES = [
+    'bookstore-5',
+    'beauty-3',
+    'cobalt-1',
+    'generic-6',
+    'enark-1',
+    'bewise-1',
+    'generic-16',
+    'generic-12',
+    'clean-2',
+    'generic-8',
+    'avantgarde-3',
+    'nano-2',
+    'graphene-2',
+    'notes-3',
+    'generic-2',
+    'avantgarde-6',
+];
+
+//---------------------------------------------------------
+// Utils
+//---------------------------------------------------------
+
+/**
+ * Return an array containing the red, green and blue components
+ * of a hex color in decimal base.
+ *
+ * @param {string} color
+ * @returns {number[]}
+ */
+function getRGBDecimal(color) {
+    const rgb = [1, 3, 5].map((idx) => parseInt(color.slice(idx, idx+2), 16))
+    return rgb;
+}
```

Comment:
There is already utils to manipulate colors, here you can use `convertCSSColorToRgba`